app/utils/bitschema_utils.py

```diff
@@ -0,0 +1,599 @@
+from enum import Enum, unique
+
+
+@unique
+class Timezone(Enum):
```

Comment:
Is this required? There should be something inbuilt to do the same thing. I know there is in java.

Reply:
I'm open to refactoring this Timezone enum to a more efficient way if someone can pointed out an inbuilt package that is available out there. We can try to refactor the Timezone in the BIT repo first instead of here to see if it works coz BIT schema already the tests written for the endpoints that are using this Timezone enum.

Reply:
https://www.iana.org/time-zones

Reply:
Also [this](https://stackoverflow.com/questions/44836984/is-there-a-reason-for-java-time-zoneid-not-including-an-enum-of-zoneids#:~:text=3%20Answers&text=I%20believe%20it%27s%20because%20the,change%20regardless%20of%20Java%20version.&text=If%20the%20timezone%20data%20is,leaving%20the%20API%20%22incomplete%22.) because we use java for the android repo

Reply:
Instead of this enum we could do [this](https://docs.python.org/3/library/zoneinfo.html)

Reply:
@epicadk , unfortunately, this `zoneinfo` module only available for Python >= 3.9. 
<img width="1030" alt="Screen Shot 2021-04-22 at 10 38 41 am" src="https://user-images.githubusercontent.com/29667122/115639247-7415cc80-a357-11eb-9b06-1c8583fc07a0.png">


MS backend still using Python 3.6 for gh-action, so the gh action build/test will fail unless we change this.
<img width="609" alt="Screen Shot 2021-04-22 at 10 41 35 am" src="https://user-images.githubusercontent.com/29667122/115639407-d8389080-a357-11eb-91a8-8ff692acd309.png">

Also, this enum is currently used in BIT frontend in [timezones.js](https://github.com/anitab-org/bridge-in-tech-web/blob/develop/src/timezones.js) as a constant so that we can provide the dropdown options to user to prevent them inserting timezone that doesn't fall within the accepted value of backend timezone enum (which is according to the [pytz list](https://gist.github.com/heyalexej/8bf688fd67d7199be4a1682b3eec7568). Do you have any suggestions how we do this if we refactor backend to `zoneinfo`? What should we use as constant at the frontend?

Reply:
We'll run a matrix build. I'll create an issue.

Reply:
@mtreacy002 can you know use python 3.9 and use zoneinfo?